pkg/api/v1/defaults.go

```diff
@@ -82,6 +82,24 @@ func SetDefaults_FloppyTarget(obj *FloppyTarget) {
 	}
 }
 
+func SetDefaults_DiskTarget(obj *DiskTarget) {
+	if obj.Bus == "" {
+		obj.Bus = "virtio"
+	}
+	if obj.Device == "" {
+		switch obj.Bus {
+		case "ide":
+			obj.Device = "hda"
```

Comment:
That does not look right.  That would give every disk the same device name hint.